visualization/app/codeCharta/ui/searchBar/searchBar.component.ts

```diff
@@ -46,6 +46,14 @@ export class SearchBarController implements BlacklistSubscriber {
 		this.resetSearchPattern()
 	}
 
+	public isSearchPatternEmpty() {
+		return this._viewModel.searchPattern === ""
+	}
+
+	public isPatternInBlackList() {
+		this.updateViewModel(this.settingsService.getSettings().fileSettings.blacklist)
```

Comment:
the function name does not fit to what it does!

Reply:
And why do we need this function? whenever the blacklist changes we update our viewModel..

Reply:
class was refactored.